srcpkgs/step-certificates/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'step-certificates'
+pkgname=step-certificates
+version=0.19.0
+revision=1
+wrksrc=certificates-$version
+build_style=go
+go_import_path=github.com/smallstep/certificates
+go_package="$go_import_path/cmd/step-ca
+ $go_import_path/cmd/step-awskms-init
+ $go_import_path/cmd/step-cloudkms-init
+ $go_import_path/cmd/step-pkcs11-init
+ $go_import_path/cmd/step-yubikey-init"
+go_ldflags="-w -X main.Version=$version"
+hostmakedepends="pkg-config"
+makedepends="pcsclite-devel"
+short_desc="Private CA and ACME server"
+maintainer="Michael Aldridge <maldridge@voidlinux.org>"
+license="Apache-2.0"
+homepage="https://smallstep.com/certificates/"
+changelog="https://github.com/smallstep/certificates/blob/master/CHANGELOG.md"
```

Comment:
i think the convention is https://raw.githubusercontent.com/smallstep/certificates/master/CHANGELOG.md (same for the other template